site/_data/i18n/tags.yml

```diff
@@ -211,3 +211,12 @@ chrome70:
 
 insider:
   en: Chrome Dev Insider
+
+web-vitals:
+  en: Web Vitals
+  es: Web Vitals
```

Comment:
The `en` value is the default when there isn't a i18n version of a given string, so there's no need to include language keys if the values are the same as `en`.

Reply:
ah, I just copied over from the web.dev tag but that makes sense. I'll remove the other keys